src/main/java/com/powsybl/openloadflow/ac/VoltageMagnitudeInitializer.java

```diff
@@ -98,6 +98,7 @@ public InitVmBusEquationTerm(LfBus bus, VariableSet<InitVmVariableType> variable
             variables = new ArrayList<>(neighbors.size());
             der = new TDoubleArrayList(neighbors.size());
             double bs = 0; // neighbor branches susceptance sum
+            double lowImpedanceThreshold = new LfNetworkParameters().getLowImpedanceThreshold();
```

Comment:
That is not correct: you have to carry the `LfNetworkParameters`  object until here.